From c2c0e7977b2311b61a4756663a4039c4c8a98d71 Mon Sep 17 00:00:00 2001
From: Trio Patch Bot <patch-bot@users.noreply.github.com>
Date: Wed, 4 Mar 2026 21:55:24 +0100
Subject: [PATCH] Update G7SensorKit (show reading timestamp with seconds)

---
 .gitmodules | 2 +-
 G7SensorKit | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index ca2bf7849..83cc6918b 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -20,7 +20,7 @@
 	branch = trio
 [submodule "G7SensorKit"]
 	path = G7SensorKit
-	url = https://github.com/loopandlearn/G7SensorKit.git
+	url = https://github.com/cachrisman/G7SensorKit.git
 	branch = trio
 [submodule "OmniKit"]
 	path = OmniKit
diff --git a/G7SensorKit b/G7SensorKit
index 0c0930500..8b212ce48 160000
--- a/G7SensorKit
+++ b/G7SensorKit
@@ -1 +1 @@
-Subproject commit 0c093050084b63d7af0dc99307dba09239eb3180
+Subproject commit 8b212ce48e7de768e918481e4480bcd204754c09
-- 
2.50.1 (Apple Git-155)

