From 9cfde4a8e68b4eea43998302f3274bcdb68a01e9 Mon Sep 17 00:00:00 2001
From: Trio Patch Bot <patch-bot@users.noreply.github.com>
Date: Tue, 14 Apr 2026 00:24:02 +0200
Subject: [PATCH] Update G7SensorKit (show reading timestamp with seconds)

---
 .gitmodules | 2 +-
 G7SensorKit | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 640982671..c344f7474 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -15,7 +15,7 @@
 	url = https://github.com/loopandlearn/OmniBLE.git
 [submodule "G7SensorKit"]
 	path = G7SensorKit
-	url = https://github.com/loopandlearn/G7SensorKit.git
+	url = https://github.com/cachrisman/G7SensorKit.git
 [submodule "OmniKit"]
 	path = OmniKit
 	url = https://github.com/loopandlearn/OmniKit.git
diff --git a/G7SensorKit b/G7SensorKit
index 4d0780db0..b791cf56c 160000
--- a/G7SensorKit
+++ b/G7SensorKit
@@ -1 +1 @@
-Subproject commit 4d0780db06c7c95b3a3bf3cdb2f2838d521e411a
+Subproject commit c1da27f28f77098e88af7f318bfbe300d68cb7e6
-- 
2.50.1 (Apple Git-155)

